content/reference/api/hub/latest.yaml

```diff
@@ -6,112 +6,104 @@ info:
 
       ### Scopes
 
-      For each scope grouping (in this case "repo"), you only need to define 1 scope as any lower 
-      scopes are assumed. For example: If you define `repo:write`, the API assumes the scope of both 
-      `repo:read` *and* `repo:public_read` as well. If you were to define both `repo:write` *and* 
-      `repo:read`, then `repo:read` is assumed by `repo:write` and ignored.
+      For each scope grouping (in this case "repo"), you only need to define 1 scope as any lower scopes are assumed.
+      For example: If you define `repo:write`, the API assumes the scope of both `repo:read` *and* `repo:public_read` as well.
+      If you were to define both `repo:write` *and* `repo:read`, then `repo:read` is assumed by `repo:write` and ignored.
 
-      ***Treat your personal access token like your password and keep it secret. You cannot retrieve 
-      your token after it is generated.***
+      ***Treat your personal access token like your password and keep it secret. You cannot retrieve your token after it is generated.***
   - name: audit-logs
     x-displayName: Audit Logs
     description: |
-      The Audit Logs API endpoints allow you to query audit log events across a
-      namespace.
+      The Audit Logs API endpoints allow you to query audit log events across a namespace.
 
       For more information, see [Audit Log](https://docs.docker.com/admin/organization/activity-logs/).
   - name: org-settings
     x-displayName: Org Settings
     description: |
-      The Org Settings API endpoints allow you to manage your organization's
-      settings.
+      The Org Settings API endpoints allow you to manage your organization's settings.
   - name: repositories
     x-displayName: Repositories
     description: |
-      The repository endpoints allow you to access your repository's
-      tags.
+      The repository endpoints allow you to access your repository's tags.
+  - name: orgs
+    x-displayName: Organizations
+    x-audience: public
+    description: |
+      The organization endpoints allow you to interact with and manage your organizations.
+
+      For more information, see [Organization administration overview](https://docs.docker.com/admin/organization/).
+  - name: groups
+    x-displayName: Groups (Teams)
+    x-audience: public
+    description: |
+      The groups endpoints allow you to manage your organization's teams and their members.
+
+      For more information, seee [Create and manage a team](https://docs.docker.com/admin/organization/manage-a-team/).
+  - name: invites
+    x-displayName: Invites
+    x-audience: public
+    description: |
+      The invites endpoints allow you to manage invites for users to join your Docker organization.
+
+      For more information, see [Invite members](https://docs.docker.com/admin/organization/members/#invite-members).
```

Comment:
I am not able to see any of these in the preview: https://deploy-preview-21615--docsdocker.netlify.app/reference/api/hub/latest/#tag/repositories

Reply:
Fixed